packages/ui-kit/src/design-system.ts

```diff
@@ -288,8 +288,9 @@ export const dimensions = {
 
 export const typography = {
   fontFamilies: {
-    primary: "'Roboto', sans-serif",
+    primary: "'Inter var', sans-serif",
     code: "'Roboto Mono', monospace",
+    stringLiteral: "'Inter var'",
```

Comment:
you may choose monospace as the generic family type fallback here.  In doubt that matches the use case better.